defaults/main.yml

```diff
@@ -27,7 +27,7 @@ atom_csrf_protection: "no"
 #   - For RH/Centos 70,71,72 and 73 are available. Default is 72
 #   - Ubuntu 14.04 uses php 5, 16.04 7.0 and 18.04 7.2
 #
-# atom_php_version: 72
+atom_php_version: 72
```

Comment:
I was wondering if uncommenting `atom_php_version` in defaults could have undesired side efects. It looks like the role depends on it to be undefined to "automagically" set up the php version to install:

```
> grep -nr atom_php_version . 

./vars/RedHat.yml:1:php_version: "{{ atom_php_version|default('72') }}" # Without dots. 70 and 71 are also available
./vars/CentOS.yml:1:php_version: "{{ atom_php_version|default('72') }}" # Without dots. 70 and 71 are also available
./vars/Ubuntu-18.04.yml:1:php_version: "{{ atom_php_version|default('7.2') }}"
./vars/Ubuntu-14.04.yml:1:php_version: "{{ atom_php_version| default('5') }}"
./vars/Ubuntu-16.04.yml:1:php_version: "{{ atom_php_version|default('7.0') }}"
./defaults/main.yml:30:# atom_php_version: 72
```

If the default is uncommented all the above statements will become ineffective